engine-spring/core/src/main/java/org/operaton/bpm/engine/spring/components/registry/ActivitiStateHandlerRegistration.java

```diff
@@ -32,13 +32,13 @@
  */
 public class ActivitiStateHandlerRegistration {
 	private Map<Integer, String> processVariablesExpected = new ConcurrentHashMap<Integer, String>();
-	private Method handlerMethod;
-	private Object handler;
-	private String stateName;
-	private String beanName;
+  private final Method handlerMethod;
```

Comment:
This change caused an uneven layout in the class itself

Reply:
tab -> 2 spaces